versions.tf

```diff
@@ -14,14 +14,18 @@
  * limitations under the License.
  */
 
-
 terraform {
   required_version = ">= 0.12"
   required_providers {
-    google   = "~> 3.7"
-    helm     = "~> 0.10"
-    null     = "~> 2.0"
-    template = "~> 2.0"
-    random   = "~> 2.0"
+    google      = "~> 3.7"
```

Comment:
Is there a specific reason this is in the "3.7" format vs. "3.7.0" in the examples file?

Reply:
Putting 3.7 in the module to make it more flexible for consumers. But put 3.7.0 in example or config that consumes this module to be more specific.